internal/api/otp.go

```
@@ -83,6 +88,10 @@ func (a *API) SmsOtp(w http.ResponseWriter, r *http.Request) error {
 	if err := json.Unmarshal(body, params); err != nil {
 		return badRequestError("Could not read sms otp params: %v", err)
 	}
+	if params.Phone != "" && !sms_provider.IsValidMessageChannel(params.Channel, *config) {
```

Can we return an explicit error if the channel is not supported or configured?